APE0.rst

```diff
@@ -0,0 +1,360 @@
+users, and even potential participants in the broader astronomical and
+scientific software community. All have a vested interest in the success of the
+Project, so all governance in the end belongs to this community in its entirety.
+The governance Charter described below provides levels of delegation of that
+authority, particularly to the Voting Members and the Coordination Committee.
+
+The Coordination Committee
+--------------------------
+
+Composition
+^^^^^^^^^^^
+The Coordination Committee is a committee that is composed of four members. To
+represent a broad set of interests, the Coordination Committee should have no
+more than two members from a particular employer. The Ombudsperson (described in
+Section 4) and the Coordination Committee will work to maintain this composition
+following the processes laid out in this Charter.
+
+Role
+^^^^^^^
+The Coordination Committee’s role is to facilitate consensus in the Project and
+coordinate Project-spanning efforts. As such, the Coordination Committee shall
+work to:
+
+* Maintain the quality and stability of the Astropy ecosystem of packages for
+  the community,
+* Facilitate the development of resources that improve accessibility and
+  understanding of the Astropy ecosystem,
+* Make contributing as accessible, inclusive, and sustainable as possible,
+* Oversee the financial and legal status of the project,
+* Lead and oversee the decision-making process for APEs and other community
+  decisions,
+* Seek consensus among the Astropy community, particularly before acting in a
+  formal capacity,
+* Act as the decision-making body where all other methods for achieving
+  consensus have failed,
+* Operate openly and provide transparency and opportunity for community input
+  into its activities.
+
+Powers
+^^^^^^
+The Coordination Committee has broad authority to make decisions about the
+Project. For example, they can:
+
+* Accept or reject APEs and updates to APEs (except this one - see below);
+* Enforce or update the Project's code of conduct
+* Manage any project assets (including financial and physical assets)
+* Delegate parts of their authority to other subcommittees or processes
+
+However, they cannot modify the Governance of the Project (i.e., this document),
+except during the initial adoption of this Charter as specified herein.
+
+The Coordination Committee has broad authority, which it seeks to exercise as
+rarely as possible by working towards consensus. The Coordination Committee
+should look for ways to use these powers as little as possible, striving to
+involve the community whenever possible for decisions, and delegating
+responsibilities.
+
+When delegating its power by creating roles or making appointments the
+Coordination Committee must solicit community feedback for no less than two
+weeks.
+
+Making Decisions
+^^^^^^^^^^^^^^^^
+To use its powers, the Coordination Committee must first attempt to reach a
```

Comment:
I'm mildly uncomfortable with references here and in a couple other places to the CoCo's "powers".  Even though this document stresses that one of the main purposes of the CoCo is to seek consensus, rather than wield "power" authoritatively, the use of the word seems to undermine that a bit.

What about we instead speak of the CoCo's "prerogatives"?  It's a little more legally precise, referring to exclusive rights (and of course can be used authoritatively), but to me it somehow sounds a little more precise and limited in scope.

Reply:
So in this sentence you could write, "Before exercising its prerogatives, the Coordination Committee must ..."

Reply:
Good points!  I like "prerogatives" although I'm a bit concerned it's a bit too obscure of a word particularly given plenty of non-native  English speakers in the Project. But I agree with the main point here, so if no one else has a better word I think it's better than "powers".

Reply:
Maybe this will help? https://www.thesaurus.com/browse/power

Reply:
Hmm.  The "regency of the CoCo"?  Or "the imperium of the CoCo"? :wink: 

More seriously, the only one I found that might be better is "authority" or possibly "privilege". @embray (or @pllim!) do either of those strike you as clearer?  If not I'm fine with just accepting "prerogative"

Reply:
I was thinking more of "capacity" but I don't know how to make it sound nice in that sentence, so I'll defer to those with stronger opinions. 😬

Reply:
Oh, and I guess maybe @embray suggested "rights" later on.  I think I kinda like "priviledge" more compared to that because it makes clear that it's something the community has given the CoCo not something that comes "automatically".  But I'm open to @embray's thoughts!

Reply:
Privilege is also fine.

Reply:
(to close the loop, I've tried to use a mix of "privilege" and "authority", because in some places "authority" was the only grammatically sensible choice)